<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partMainConsole</class>
 <widget class="QWidget" name="partMainConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="txtConsole">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,0">
        <item>
         <widget class="ComboBox" name="cboCommand">
          <property name="styleSheet">
           <string notr="true">padding-top: 3;
padding-bottom: 3;
padding-right: 8;</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>10</number>
          </property>
          <property name="maxCount">
           <number>25</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertPolicy::InsertAtTop</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdCommandSend">
          <property name="toolTip">
           <string>Send command</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/send_1.png</normaloff>:/images/send_1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdClearConsole">
          <property name="toolTip">
           <string>Clear console</string>
          </property>
          <property name="icon">
           <iconset resource="../../images.qrc">
            <normaloff>:/images/erase_1.png</normaloff>:/images/erase_1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/combobox.h</header>
   <slots>
    <signal>returnPressed()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdCommandSend</sender>
   <signal>clicked()</signal>
   <receiver>partMainConsole</receiver>
   <slot>onSendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdClearConsole</sender>
   <signal>clicked()</signal>
   <receiver>partMainConsole</receiver>
   <slot>onClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboCommand</sender>
   <signal>returnPressed()</signal>
   <receiver>partMainConsole</receiver>
   <slot>onSendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClearClicked()</slot>
  <slot>onSendClicked()</slot>
 </slots>
</ui>
